docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+app:
+  image: chuckharmston/universal-search-recommendation-app
```

Comment:
Note: this exists at `chuckharmston/` only until I get them created under the Mozilla org (request in progress).